radix-engine-tests/tests/execution_trace.rs

```diff
@@ -574,6 +573,7 @@ fn test_worktop_changes() {
                     ]
                     .into()
                 )),
+                WorktopChange::Take(ResourceSpecifier::Amount(fungible_resource, 100.into())),
```

Comment:
Out of interest - why has this order changed? Shouldn't it be deterministic if we're using a vec?

Reply:
I guess it might be deterministic based on some BTreeMap of some objectid hash which has changed due to the addressing changes?

Reply:
Not sure, this seems to keep on changing every once in a while and haven't had a chance to dig into why.